test/src/index/transport/testBrowserstackTransport.js

```diff
@@ -341,7 +341,80 @@ describe('BrowserstackTransport', function () {
     assert.strictEqual(transport.buildId, '123-567-89');
 
   });
+  
+  it('test create Transport for Browserstack - Automate TurboScale', async function() {
+    const client = NightwatchClient.client({
+      webdriver: {
+        host: 'hub-cloud.browserstack-ats.com',
+        port: 443,
+        start_process: true
+      },
+      desiredCapabilities: {
+        'browserstack.user': 'test-access-user',
+        'browserstack.key': 'test-access-key',
+        browserName: 'chrome'
+      }
+    });
 
+    nock('https://hub-cloud.browserstack-ats.com')
+      .post('/wd/hub/session')
+      .reply(201, function (uri, requestBody) {
+        return {
+          value: {
+            sessionId: '1352110219202',
+            capabilities: requestBody.capabilities
+          }
+        };
+      });
+    
+    nock('https://api.browserstack.com')
+      .get('/automate-turboscale/v1/builds?status=running')
+      .reply(200, {
+        'builds': [
+          {
+            name: 'nightwatch-test-build',
+            hashed_id: '123-567-89'
+          },
+          {
+            name: 'test-build'
+          }
+        ]
+      });
+    assert.ok(client.transport instanceof AutomateTurboScale);
+    assert.strictEqual(client.settings.webdriver.host, 'hub-cloud.browserstack-ats.com');
+    assert.strictEqual(client.settings.webdriver.default_path_prefix, '/wd/hub');
+    assert.strictEqual(client.settings.webdriver.ssl, true);
+
+    const {transport} = client;
+    assert.ok(transport instanceof SeleniumRemote);
+
+    let result = await transport.createSession({argv: undefined, moduleKey: ''});
+    result.sessionId = '1234567';
+    client.emit('nightwatch:session.create', result);
+
+    assert.strictEqual(transport.username, 'test-access-user');
+    assert.strictEqual(transport.accessKey, 'test-access-key');
+    assert.strictEqual(client.settings.webdriver.start_process, false);
+      
+    nock('https://api.browserstack.com')
+      .get('/automate-turboscale/v1/sessions/1234567')
+      .reply(200, {
+        status: 'done'
+      });
+    nock('https://api.browserstack.com')
+      .patch('/automate-turboscale/v1/sessions/1234567', {
+        status: 'passed',
+        reason: ''  
+      })
+      .reply(200, {});
```

Comment:
Also ensure that both these nocks are actually being called by using a counter flag. See: https://github.com/nightwatchjs/nightwatch/blob/main/test/src/index/transport/testBrowserstackTransport.js#L265